qaframework-bdd-tests/src/test/resources/features/refapp-2.x/07-visit/visitTypes.feature

```diff
@@ -1 +1,33 @@
-Feature: Visit types Management
\ No newline at end of file
+Feature: Visit types Management
+
+  Background:
+    Given User clicks on the system administration link on the home page
```

Comment:
what happens when a user click on the system administration link?

Reply:
next....**User clicks on the manage Visit Types link on the administration page** .....any suggestion @kdaud

Reply:
do you mind taking a look at the existing feature files user stories?